src/test/incremental/const-generics/hash-tyvid-regression-1.rs

```diff
@@ -0,0 +1,15 @@
+// revisions: cfail
```

Comment:
do we need revisions at all if there is only one?

Reply:
if you dont have a revision the testing stuff shouts at you saying `Incremental tests require revisions.`

Reply:
lol ok. I guess the alternative would be to move the test to ui tests and add flags, but I don't see why that would be better.